DigitalLearningSolutions.Data/DataServices/SelfAssessmentDataService/CompetencyDataService.cs

```diff
@@ -0,0 +1,532 @@
+﻿namespace DigitalLearningSolutions.Data.DataServices.SelfAssessmentDataService
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Dapper;
+    using DigitalLearningSolutions.Data.Models.Frameworks;
+    using DigitalLearningSolutions.Data.Models.SelfAssessments;
+    using Microsoft.Extensions.Logging;
+    using AssessmentQuestion = DigitalLearningSolutions.Data.Models.SelfAssessments.AssessmentQuestion;

```

Comment:
What does this do?

Reply:
A "using alias". Allows you to alias something so that it is clear which one you are using.

In this case, there is an AssessmentQuestion in `DigitalLearningSolutions.Data.Models.Frameworks` and in `DigitalLearningSolutions.Data.Models.SelfAssessments` so we have to specify that the AssessmentQuestion we want to use further down the file is the SelfAssessments one.

If we needed to use both, we would alias them both, e.g. as `using FrameworksAssessmentQuestion = ...` and `using SelfAssessmentQuestion = ...`